lib/blacklight/xml_parser.rb

```diff
@@ -113,7 +113,8 @@ def self.get_master_parent(pre_data, parents_ids, parent_id)
   # non-metadata file.
   ##
   def self.iterate_files(zipfile)
-    files = zipfile.glob("*/**/**")
+    files = zipfile.entries.select(&:file?)
```

Comment:
Some thoughts: @bfcoder -- have you tried it with the same export that @nick-benoit14 is using? Also, bundler should be ensuring you're all on the same version of ruby-zip, but it might be worth double-checking that. Lastly, you should make sure you're all doing this exactly the same -- are @bfcoder and @nick-benoit14 both checking `zipfile.glob` at the same point in the code, via `byebug`? A `zipfile.inspect` might be helpful, too (to help catch any unexpected differences, for instance).